app/views/partials/banner.njk

```diff
@@ -2,9 +2,9 @@
 <div class="app-banner app-banner--warning" data-module="app-banner">
   <div class="govuk-width-container">
     <p class="govuk-body">
-      This is not a real service, and is used for testing purposes only.
+      This is an internal development app.
       <br>
-      See the <a class="govuk-link govuk-link--inverse" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> for recommended guidance.
+      To learn how to use the GOV.UK Design System in your project, see [Get started](https://design-system.service.gov.uk/get-started/).
```

Comment:
I don't think there you can use Markdown in this context, so this link should be hardcoded:

```suggestion
      To learn how to use the GOV.UK Design System in your project, see <a class="govuk-link govuk-link--inverse" href="https://design-system.service.gov.uk/get-started/">Get started</a>.
```

Reply:
Ah interesting, did not know that - thanks, @lfdebrux 👍🏻